org.eclipse.lsp4e.jdt/src/org/eclipse/lsp4e/jdt/internal/LSJavaHoverProvider.java

```diff
@@ -9,7 +9,7 @@
  * Contributors:
  *  - Martin Lippert (Pivotal Inc.) - Initial implementation
  *******************************************************************************/
-package org.eclipse.lsp4e.jdt;
+package org.eclipse.lsp4e.jdt.internal;
```

Comment:
shall we not rather do this move in different PR?

Reply:
Yes, we can do that. I'll revert moving classes. I did that because had to export a package and didn't mean to export more than needed... Wonder if perhaps we could have a preference page for LSP4E JDT bit and expose the preference I'm introducing on that new preference page... This way we can completely avoid exposing anything as API...